packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.scss

```diff
@@ -113,14 +112,13 @@ $ms-DocumentCardActions-verticalPadding: 4px;
 .views {
   @include text-align(right);
   line-height: $ms-DocumentCardActions-actionSize;
-
-  :global(.ms-Icon) {
-    @include ms-margin-right(4px);
-    font-size: $ms-icon-size-m;
-    vertical-align: top;
-  }
 }
 
+.viewsIcon {
```

Comment:
For the reviewer: this is the only real update - I moved the ":global" out since .ms-Icon is no longer used anymore inside the Icon component.